packages/gensx-storage/package.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "@gensx/storage",
+  "version": "0.0.1",
+  "description": "Cloud storage, blobs, sqlite, and vector database providers/hooks for GenSX.",
+  "type": "module",
+  "main": "./dist/index.js",
+  "module": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "engines": {
+    "node": ">=18.0.0",
+    "pnpm": ">=9.0.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/gensx-inc/gensx.git",
+    "directory": "packages/gensx-storage"
+  },
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js"
+    },
+    "./blob": {
+      "types": "./dist/blob/index.d.ts",
+      "import": "./dist/blob/index.js"
+    }
+  },
+  "scripts": {
+    "build": "tsc -p tsconfig.build.json",
+    "clean": "rm -rf dist",
+    "test": "vitest run --coverage",
+    "test:watch": "vitest watch",
+    "test:types": "tsc --noEmit",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix"
+  },
+  "keywords": [
+    "gensx",
+    "storage",
+    "blobs",
+    "sqlite",
+    "vectorDB"
+  ],
+  "files": [
+    "dist"
+  ],
+  "author": "GenSX Team",
+  "license": "Apache-2.0",
+  "devDependencies": {
+    "@types/node": "catalog:packages",
+    "@vitest/coverage-istanbul": "catalog:",
+    "vitest": "catalog:"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@gensx/core": "workspace:*",
+    "zod": "catalog:",
+    "zod-to-json-schema": "catalog:"
```

Comment:
We don't use zod in the storage package as far as I can see.

Reply:
yep good point, removing